packages/earljs/src/Expectation.ts

```diff
@@ -40,65 +44,125 @@ export class Expectation<T> {
 
   // validators
 
-  /** Does deep "smart" equality check. */
+  /**
+   * Performs a recursive equality check. Objects are equal if their fields
+   * are equal and they share the same prototype.
+   * @param value value to check against.
+   */
   toEqual(value: T): void {
     toEqual(this.getControl(), value)
   }
 
-  /** Like toEqual but without type checking. */
+  /**
+   * Performs a recursive equality check. Objects are equal if their fields
+   * are equal. Object prototypes are ignored.
+   * @param value value to check against.
+   */
   toLooseEqual(value: any): void {
     toLooseEqual(this.getControl(), value)
   }
 
-  /** Checks referential equality */
+  /**
```

Comment:
It would be cool to move these comments to API reference docs. It would be best to automatically write API reference based on these docs 😅